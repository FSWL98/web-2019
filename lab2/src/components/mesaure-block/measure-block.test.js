import React from "react";
import MeasureBlock from "./measure-block";
import renderer from 'react-test-renderer';

const measurements = [
    {
        color: "gray",
        name: 'Name 1',
        text: "text 1"
    },
    {
        color: "blue",
        name: 'Name 2',
        text: "text 2"
    }
];

it("Empty MeasureBlockItem", () => {
    const component = renderer.create(<MeasureBlock measurements={[]} />);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});

it("Filled MeasureBlockItem", () => {
    const component = renderer.create(<MeasureBlock measurements={measurements}/>);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});
