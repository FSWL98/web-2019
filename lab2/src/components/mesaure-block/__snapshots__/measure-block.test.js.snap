// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Empty MeasureBlockItem 1`] = `
<div
  className="container"
/>
`;

exports[`Filled MeasureBlockItem 1`] = `
<div
  className="container"
>
  <div
    className="measure-block-item measure-block-item--gray d-flex mt-md-1 justify-content-between row"
  >
    <div
      className="col"
    >
      Name 1
    </div>
    <div
      className="d-flex justify-content-end font-weight-light col"
    >
      text 1
    </div>
  </div>
  <div
    className="measure-block-item measure-block-item--blue d-flex mt-md-1 justify-content-between row"
  >
    <div
      className="col"
    >
      Name 2
    </div>
    <div
      className="d-flex justify-content-end font-weight-light col"
    >
      text 2
    </div>
  </div>
</div>
`;
